### PR TITLE
Add label checks for labels used in k8s 1.14

### PR DESCRIPTION
**Pull request by MarcelMue**
Labels change with k8s 1.14 - this is basically the same PR as https://github.com/giantswarm/gsctl/pull/352 for gsctl. 

I tested this briefly on gauss and everything seems to be working fine.

### CODE REVIEW

File: src/components/cluster/detail/view.js [hunk 1]
```diff
@@ -193,7 +193,10 @@ class ClusterDetailView extends React.Component {
       var workers = 0;
       nodes.forEach(node => {
         if (Object.keys(node).includes('labels')) {
-          if (node.labels['role'] != 'master') {
+          if (
+            node.labels['role'] != 'master' &&
+            node.labels['kubernetes.io/role'] != 'master'
+          ) {
```
Comment (pipo02mix) [inline src/components/cluster/detail/view.js]: Why do you need to check both labels? I dont see same logic in gsctl 🤔

Comment (MarcelMue) [reply src/components/cluster/detail/view.js]: I added effectively the same logic in gsctl here:
https://github.com/giantswarm/gsctl/pull/352/files

The reason is that label name changed with k8s 1.14.

Comment (oponder) [reply src/components/cluster/detail/view.js]: So basically it can be either `role` or `kubernetes.io/role` depending on the cluster version right?

Comment (MarcelMue) [reply src/components/cluster/detail/view.js]: Yes, exactly.

**Pull request merged**